### PR TITLE
FIX ESlint NavBar.js

### PR DESCRIPTION
**Pull request by sarovin**

### CODE REVIEW

**Comment by aksonov** [comment]
Hm... Why NavBar.contextTypes are declared before NavBar declaration?

**Comment by charpeni** [comment]
@sarovin Please refer to the [Airbnb React/JSX Style Guide](https://github.com/airbnb/javascript/tree/master/react#ordering) specially ordering section and _How to define propTypes, defaultProps, contextTypes, etc..._

**Comment by sarovin** [comment]
@charpeni it's good?

**Comment by aksonov** [comment]
@sarovin What is wrong with existing NavBar?

**Comment by sarovin** [comment]
@aksonov i have fixed some Eslint error.

**Comment by sarovin** [comment]
There is also
` 305:27  error  'NavigationState' is not defined                             no-undef`

File: src/NavBar.js [hunk 1]
```diff
@@ -256,5 +104,272 @@ const styles = StyleSheet.create({
...
+        console.warn(
+          'Both onLeft and leftTitle/leftButtonImage must be specified for the scene: ' +
+          state.name
+        );
+      }
+    }
+    return tryRender(this.props);
+  }
+
+  renderTitle(childState: NavigationState, index:number) {
```
Comment (charpeni) [inline src/NavBar.js]: Maybe this would fix the issue :
```javascript
renderTitle(/* NavigationState */ childState, index: number)
```

**Comment by sarovin** [comment]
I have fixed NavBar height for android...

**Comment by charpeni** [comment]
FYI 12 eslint errors remaining.

Can you move 355c03c _Fix NavBar height for Android_ to a different PR?

**Comment by sarovin** [comment]
It's a problem for me..

**Comment by sarovin** [comment]
#625

**Pull request closed without merging**